webapp/channels/src/components/admin_console/ip_filtering/add_edit_ip_filter_modal.tsx

```diff
@@ -111,7 +111,7 @@ export default function IPFilteringAddOrEditModal({onExited, onSave, existingRan
                                 values={{
                                     link: (msg) => (
                                         <ExternalLink
-                                            href='https://docs.mattermost.com/guides/cloud-workspace-management.html'
+                                            href='https://docs.mattermost.com/manage/cloud-ip-filtering.html'
```

Comment:
Drive by review :car: : Should a perma link be used here similar to how the links in the system console work? See also https://docs.google.com/spreadsheets/d/1O601H_A0IM8pR3FOfue29_C8flGV7xK3ts-tJUVDHHg/edit#gid=607040777

Reply:
This is the link I was given, I believe it is a permalink https://mattermost.atlassian.net/browse/CLD-6778?focusedCommentId=159624

Reply:
cc @cwarnermm to confirm

Reply:
Apologies, @nickmisasi! Copy/pasta error. Here's the correct link: ``https://mattermost.com/pl/cloud-ip-filtering``